Description: Enable metadata_csum by default
 For debian testing let's be more aggressive about testing the
 metadata_csum feature.
From: Theodore Y. Ts'o <tytso@mit.edu>

--- e2fsprogs-1.43.orig/misc/mke2fs.conf.in
+++ e2fsprogs-1.43/misc/mke2fs.conf.in
@@ -11,7 +11,7 @@
 		features = has_journal
 	}
 	ext4 = {
-		features = has_journal,extent,huge_file,flex_bg,uninit_bg,64bit,dir_nlink,extra_isize
+		features = has_journal,extent,huge_file,flex_bg,metadata_csum,64bit,dir_nlink,extra_isize
 		inode_size = 256
 	}
 	small = {
